frontend_app_thread/src/container/PopupCreateThread.jsx

```diff
@@ -76,50 +96,33 @@ class PopupCreateThread extends React.Component {
     }
   })
 
+  sendGlobalFlashMessage = msg => GLOBAL_dispatchEvent({
+    type: CUSTOM_EVENT.ADD_FLASH_MSG,
+    data: {
+      msg: msg,
+      type: 'warning',
+      delay: undefined
+    }
+  })
+
   handleValidate = async () => {
-    const { config, appName, workspaceId, folderId, newContentName } = this.state
+    const { props, state } = this
 
-    const fetchSaveThreadDoc = postThreadContent(config.apiUrl, workspaceId, folderId, config.slug, newContentName)
+    const fetchSaveThreadDoc = postThreadContent(
+      state.config.apiUrl, state.workspaceId, state.folderId, state.config.slug, state.newContentName
+    )
 
     const resSave = await handleFetchResult(await fetchSaveThreadDoc)
 
     switch (resSave.apiResponse.status) {
-      case 200:
```

Comment:
Have you tested if the status is 200, it will not fall into the default case and show an error message?

Reply:
you are right, it's an issue